server/parsec/cli/utils.py

```diff
@@ -103,7 +103,6 @@ def cli_exception_handler(debug: bool) -> Iterator[bool]:
         yield debug
 
     except KeyboardInterrupt:
-        click.echo("bye ;-)")
```

Comment:
So you remove this bye because it should be handled by the one line 470 in the run.py ?

Reply:
Yes exactly